__tests__/app.test.js

```diff
@@ -36,4 +35,34 @@ describe('GET /api/topics', () => {
                 expect(body.message).toBe('path not found')
             })
         })
+})
+
+describe('GET /api', () => {
+    test('responds with 200 status code', () =>{
+        return request(app).get('/api').expect(200);
+    })
+    test('returns an object describing the valid inital TEST endpoints',() => {
+        return request(app)
+            .get('/api')
+            .expect(200)
+            .then(({ body }) => {
+                const siteMap = body.mapAPI
+
+                expect(typeof siteMap).toBe("object")
+                expect(siteMap.hasOwnProperty('GET /api')).toBe(true)
+                expect(typeof(siteMap['GET /api'].description)).toBe("string")
+                expect(siteMap.hasOwnProperty('GET /api/topics')).toBe(true)
+                expect(typeof(siteMap['GET /api/topics'].description)).toBe("string")
+                expect(siteMap.hasOwnProperty('GET /api/articles')).toBe(true)
+                expect(typeof(siteMap['GET /api/articles'].description)).toBe("string")
+            })
```

Comment:
This is nice and it works.
But in its current format it means you would need to test this endpoint everytime you create a new endpoint.
To save you doing this for each task you could require in the endpoints JSON and expect that your return from the endpoint looks exactly like the JSON so that when we add to the JSON we don't have to repeat this test.